Makefile

```diff
@@ -30,10 +33,13 @@ directories:
 	@mkdir -p $(OUT_DIR) $(OBJ_DIR) $(LIB_DIR)
 
 executable: $(OBJS) $(EXECUTABLE_OBJS)
```

Comment:
Még egy apróság: a futtatható fájlt így mindig újralinkeli, akkor is, ha nem kell, mivel nincs `executable` nevű fájl. Targetnek a binárist kéne megadni ehhez a parancshoz: a `bin/bin.out`-ot. Aztán lehet egy köztes `executable` szabály, ami erre dependel, ha szükség van rá.

Reply:
Kész.